app/views/welcome/index.html.erb

```diff
@@ -227,15 +230,18 @@
     <div class="container-fluid">
       <div class="row">
         <div class="col-sm-7 col-sm-offset-4 col-md-8 col-md-offset-5 col-lg-9 col-lg-offset-2">
-          <h1>Results</h1>
-          <div id="results">
-            <% if @matching_songs %>
-              <%= render partial: "results", :locals => {:songs => @matching_songs} %>
-            <% else %>
-              <h3>Sorry, your search did not return any results. Please try a new chord combination</h3>
-            <% end %>
-            <div class="close-tab">
-              <a href="#close" class="js-closeToggle">Close</a>
+          <span class='brand-font watermark'>Chord Explorer</span>
+          <div id="backpage">
+            <div id="results">
+              <% if @matching_songs %>
+                <%= render partial: "results", :locals => {:songs => @matching_songs} %>
```

Comment:
This code doesn't hurt anything, but it also doesn't do anything -- that was our conclusion last night while working on pagination, anyway. Because we're passing an instance variable named tabs to the view, Rails somehow knows to go looking for the results partial, perhaps because there is a local variable named tabs. So experiment with removing the